routes/recordOverview.js

```diff
@@ -3,51 +3,113 @@ var linz = require('linz'),
     async = require('async'),
     clone = require('clone');
 
+
 /* GET /admin/:model/:id/overview */
 var route = function (req, res, next) {
 
-    var locals = {
-        model: req.linz.model,
-        record: clone(req.linz.record.toObject({ virtuals: true})),
-        permissions: req.linz.model.linz.formtools.permissions,
-        formtools: req.linz.model.linz.formtools
-    };
+    function transformDslToOverview (dsl, cb) {
 
-    async.series([
+        if (!(Array.isArray(dsl) && dsl.length)) {
+            return [];
+        }
 
-        function (cb) {
+        var overviewBody = [];
 
-            req.linz.model.linz.formtools.overview.summary.renderer(req.linz.record, req.linz.model, function (err, content) {
+        async.eachSeries(dsl, function (elm, callback) {
 
```

Comment:
we are passing the array (overview.body) to this function. We are looping through each element in the array. Overview.body should be an array which contains objects (fieldsets) and array (tabs)